internal/pkg/cli/interfaces.go

```diff
@@ -532,6 +532,10 @@ type wsSelector interface {
 	Workload(msg, help string) (string, error)
 }
 
+type dirFileSelector interface {
```

Comment:
I feel like we can name it `staticSouorceSelector` because of how much `StaticSources` is specifically designed for static site. `StaticSources` doesn't look like a generic selector that any clients that need to select dirs/files would just be able to use it as it.